### PR TITLE
Feature/generic agreement

### PR DESCRIPTION
**Pull request by cpiehl1**
# Jira Issue: [MX-337]

## What?
- Switching to generic agreements in giraffe

## Why?
-

## Optional checklist
- [ ] Unit tests written
- [x] Tested locally
- [x] Codescouted

[MX-337]: https://hedvig.atlassian.net/browse/MX-337?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/api/upstreams/productPricing.ts [hunk 1]
```diff
@@ -70,80 +70,16 @@ export interface AgreementType {
...
   address: AddressDto
   numberCoInsured: number
-  lineOfBusiness: string
+  squareMeters?: number
+  ancillaryArea?: number
+  yearOfConstruction?: number
+  numberOfBathrooms?: number
+  extraBuildings?: ExtraBuildingDto[]
+  isSubleted?: boolean
+  lineOfBusinessName: string
```
Comment (tobiasbexelius) [inline src/api/upstreams/productPricing.ts:82]: Is this not optional?

File: src/resolvers/uncategorised/crossSellQuotes.ts [hunk 2]
```diff
@@ -23,21 +26,24 @@ export const createCrossSellQuotes: MutationResolvers['createCrossSellQuotes'] =
...
   )
 
   const tasks = crossSellDataNames.map((name) => {
-    const coInsuredData =
-      name === 'swedishAccidentData'
-        ? { householdSize: numberCoInsured + 1 }
-        : { numberCoInsured }
     const body: CreateQuoteDto = {
       ...member,
+      // This will look a bit cleaner with generic quotes but we are basically throwing in anything that underwriter
```
Comment (tobiasbexelius) [inline src/resolvers/uncategorised/crossSellQuotes.ts:36]: I spoke with Östen and generic quotes will not be a thing. Not when writing anyway, when reading... probably? I'm not sure

**Review by tobiasbexelius — APPROVED** [review]
Looks good!

File: src/resolvers/uncategorised/crossSellQuotes.ts (hunk 2 above)
Comment (cpiehl1) [reply src/resolvers/uncategorised/crossSellQuotes.ts:36]: Ah what a shame 😞

**Pull request merged**